index.js

```diff
@@ -21,13 +22,26 @@ const client = new Client({
 });
 
 // Define target emojis and channels:
-const firstTargetEmoji = "📌";
-const secondTargetEmoji = "🔖";
-const thirdTargetEmoji = "📝";
-const deleteTodoEmoji = "✅";
-const firstTargetChannelId = process.env.TARGET_CHANNEL_ID;
-const secondTargetChannelId = process.env.SECOND_TARGET_CHANNEL_ID;
-const thirdTargetChannelId = process.env.THIRD_TARGET_CHANNEL_ID;
+const PIN_TARGET_EMOJI = "📌";
+const UNREAD_TARGET_EMOJI = "🔖";
+const TODO_TARGET_EMOJI = "📝";
+const DELETE_TODO_EMOJI = "✅";
+const PIN_TARGET_CHANNEL_ID = process.env.PIN_CHANNEL_ID;
+const UNREAD_TARGET_CHANNEL_ID = process.env.UNREAD_TARGET_CHANNEL_ID;
+const TODO_TARGET_CHANNEL_ID = process.env.TODO_TARGET_CHANNEL_ID;
+const TALKING_POINT_TARGET_CHANNEL_ID = process.env.TALKING_POINT_CHANNEL_ID;
+const GENERAL_CHANNEL_ID = process.env.GENERAL_CHANNEL_ID;
+
+const USERNAME_REGEX = /<@(\d+)>/g;
+const TODO_REGEX = /TODO/;
+const TALKING_POINT_REGEX = /(Talking point|TP)/;
+const TIME_IN_DAYS = 1000 * 60 * 60 * 24;
+const CHAR_LENGTH = 500;
+const FIVE_MINUTES = 1000 * 60 * 5;
+
+const deleteTodo = [false, false];
```

Comment:
Feel free to do this as a follow-up, but you may want to maintain a JS object such that the key is a message ID and the value is an array of two `false`  values. Also, it's a better practice to use a JS object for this anyways instead of an array, e.g. you could have an object like:
```
approvedBy = {
  author: false,
  assignee: false,
}
```
And set those, rather than index values (which have no meaning to them)

Reply:
Ack